radio/src/gui/9X/popups.cpp

```diff
@@ -139,28 +139,27 @@ void (*popupFunc)(uint8_t event) = NULL;
 #endif
 
 #if defined(NAVIGATION_MENUS)
-const char *s_menu[MENU_MAX_LINES];
+const char *popupMenuItems[POPUP_MENU_MAX_LINES];
 uint8_t s_menu_item = 0;
-uint16_t s_menu_count = 0;
-uint8_t s_menu_flags = 0;
-uint16_t s_menu_offset = 0;
-void (*menuHandler)(const char *result);
-const char * displayMenu(uint8_t event)
+uint16_t popupMenuNoItems = 0;
+uint16_t popupMenuOffset = 0;
+void (*popupMenuHandler)(const char *result);
+const char * displayPopupMenu(uint8_t event)
 {
   const char * result = NULL;
 
-  uint8_t display_count = min<uint8_t>(s_menu_count, MENU_MAX_LINES);
+  uint8_t display_count = min<uint8_t>(popupMenuNoItems, POPUP_MENU_MAX_LINES);
   uint8_t y = (display_count >= 5 ? MENU_Y - FH - 1 : MENU_Y);
   drawFilledRect(MENU_X, y, MENU_W, display_count * (FH+1) + 2, SOLID, ERASE);
   lcd_rect(MENU_X, y, MENU_W, display_count * (FH+1) + 2);
 
   for (uint8_t i=0; i<display_count; i++) {
-    lcd_putsAtt(MENU_X+6, i*(FH+1) + y + 2, s_menu[i], s_menu_flags);
+    lcd_putsAtt(MENU_X+6, i*(FH+1) + y + 2, popupMenuItems[i], 0);
```

Comment:
Of course your change is good on ARM boards, because the addresses values for FLASH and RAM cannot be the same!